server/api/models/note.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+ObjectIdSchema = Schema.ObjectId;
```

Comment:
I think this line needs to be:
```suggestion
const ObjectIdSchema = mongoose.Schema.Types.ObjectId;
```